python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/shape_lib_gen.py

```diff
@@ -562,6 +562,20 @@ def aten〇baddbmm(self: List[int], batch1: List[int], batch2: List[int], beta:
 def aten〇embedding(weight: List[int], indices: List[int], padding_idx: int = -1, scale_grad_by_freq: bool = False, sparse: bool = False) -> List[int]:
     return upstream_shape_functions.embedding(weight, indices, padding_idx, scale_grad_by_freq, sparse)
 
+def aten〇repeat(self: List[int], repeats: List[int]) -> List[int]:
```

Comment:
Please add it to the upstream torch. Please see this PR: https://github.com/llvm/torch-mlir/pull/908

Reply:
FYI, https://github.com/pytorch/pytorch/pull/79773